DotNet/Common/UnitTest/DiagnosticsTest/WindowsEventLoggerUnitTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="WindowsEventLoggerUnitTest.cs" company="CloudLibrary">
+//    Copyright (c) CloudLibrary 2016.  All rights reserved.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+namespace CloudLibrary.Common.UnitTest.DiagnosticsTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Diagnostics.CodeAnalysis;
+
+    using Common.Diagnostics;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Shared.CommonUnitTestHelper;
+    using Shared.UnitTestHelper;
+
+    /// <summary>
+    /// Windows Event Logger unit test
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    [TestClass]
+    public class WindowsEventLoggerUnitTest
+    {
+        /// <summary>
+        /// Test Windows Event Logger
+        /// </summary>
+        [TestMethod]
```

Comment:
Need put an attribute the exclude this class from code coverage. Please check other unit test code